.gitignore

```diff
@@ -7,3 +7,5 @@ build/*
 .externalNativeBuild
```

Comment:
Shouldn't you exclude `.gradle` as well? These are build files. You usually don't commit these files to the repository.

Reply:
Thanks, didn't know this. Will resolve this in the next few days.